Package.swift

```diff
@@ -10,17 +10,18 @@
 */
 
 import PackageDescription
+
 import class Foundation.ProcessInfo
 
 let swiftSettings: [SwiftSetting] = [
-    .unsafeFlags(["-Xfrontend", "-warn-long-expression-type-checking=1000"], .when(configuration: .debug)),
+    .unsafeFlags(["-Xfrontend", "-warn-long-expression-type-checking=1000"], .when(configuration: .debug))
```

Comment:
So the rule is no comma for a single array element? But a few lines below we do add the extra following comma for 2 or more elements? I actually like this.

Reply:
I feel like it's unfortunate that there's no rule to force trailing commas for *all* array literals, single- or multi-element. This way it messes with the diff when a collection goes from one to multiple elements, because of the creation of the first comma.